src/project-tests/technical-docs-tests.js

```diff
@@ -114,7 +114,23 @@ export default function createTechnicalDocsPageTests() {
                         '<a> elements ');
                 });
 
-            it('11. Each element with the class of "nav-link" should contain text that corresponds to the <header> text ' +
+            it('11. The <header> element in the navbar must come before any link (<a>) elements in the navbar.',
+                function() {
+                  const navLinks = document.querySelectorAll('#navbar a.nav-link');
+                  const header = document.querySelectorAll('#navbar header').item(0);
+                  let allAbove = true;
+
+                  navLinks.forEach(navLink => {
+                    if ( ! (header.compareDocumentPosition(navLink) & Node.DOCUMENT_POSITION_FOLLOWING)) {
```

Comment:
@no-stack-dub-sack Oops, old habit: the space between the paren and bang is something I insist on at my own company. The `!` operator is harder to miss when it isn't shoved against the paren, but I do understand that style is not very common (yet  ;-)). I will get this changed and resolve the merge conflict.

Thanks for your timely review!